sky/packages/sky/lib/src/material/tool_bar.dart

```diff
@@ -71,14 +83,7 @@ class ToolBar extends StatelessComponent {
       ),
       child: new DefaultTextStyle(
         style: sideStyle,
-        child: new Column(<Widget>[
-            new Container(
-              child: new Row(children),
-              height: kToolBarHeight
-            ),
-          ],
-          justifyContent: FlexJustifyContent.end
-        )
+        child: new IntrinsicHeight(child: new Column(columnChildren))
```

Comment:
I don't think we want to use IntrinsicHeight here.  We should use/create FlexJustification.collapse instead.  The problem is that IntrinsicHeight does more work than necessary in layout because it needs to compute the IntrinsicHeight.